app/routes/web.py

```
@@ -0,0 +1,216 @@
+from datetime import datetime
+from flask import Blueprint, render_template, request, redirect, flash, url_for, get_flashed_messages, jsonify
```

Ruff (0.8.2)

`flask.jsonify` imported but unused

Remove unused import: `flask.jsonify`

(F401)